docs/api/apiv3/paths/workspaces.yml

```
@@ -0,0 +1,95 @@
+# /api/v3/workspaces
+---
+get:
+  summary: List workspace
+  operationId: list_workspace
+  tags:
+    - Workspace
+  description: Returns a collection of workspace. The collection can be filtered via
+    query parameters similar to how work packages are filtered. In addition to the
+    provided filter, the result set is always limited to only contain workspace the
+    client is allowed to see.
+
+    Since workflows are the generic term for a number of resources like projects and portfolios, the returned
```

These should be pluralized to stay consistent with the specific workspace type resources.

Further below, there is a word mismatch and workflow is used where workspace should have been:

```suggestion
  summary: List workspaces
  operationId: list_workspaces
  tags:
    - Workspaces
  description: Returns a collection of workspaces. The collection can be filtered via
    query parameters similar to how work packages are filtered. In addition to the
    provided filter, the result set is always limited to only contain workspaces the
    client is allowed to see.

    Since workspaces are the generic term for a number of resources like projects and portfolios, the returned
```